src/styles/global/forms/forms.scss

```diff
@@ -154,3 +160,20 @@ input.form-check-input {
     }
   }
 }
+
+.validation-offset {
+  .form-control-feedback {
+    right: 40px;
+  }
+}
+.validation-offset-0 {
+  .form-control-feedback {
+    right: 10px;
+  }
+}
+
+.formly-empty-label {
```

Comment:
This seems to be the only CSS class with `formly` in its name. Maybe this is something that could be named more general? Especially in `forms.scss`. ;)